doc/release/2021-08-releases.rst

```diff
@@ -49,28 +52,140 @@ Core
 
 
 -   The information about the state of synchronous replication is now available via the
-    ``box.info.synchro`` interface (:tarantool-release:`2.7.3`, :tarantool-issue:`5191`).
+    ``box.info.synchro`` interface
+    (:tarantool-release:`2.7.3`, :tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5191`).
+
+-   Field type UUID is now part of field type ``SCALAR``
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6042`).
+
+-   Field type ``UUID`` is now available in SQL, and a new UUID can be generated
+    using the new SQL built-in function ``UUID()``
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`5886`).
+
+-   **[Breaking change]** The ``timeout()`` method of the ``net.box`` connection,
+    marked deprecated more than four years ago (in ``1.7.4``), has been dropped.
+    It negatively affected the performance of hot ``net.box` methods, such as ``call()`` and ``select()``
+    if those were called without specifying a timeout
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6242`).
+
+-   Improved ``net.box`` performance by up to 70% by rewriting hot code paths in C
+    (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6241`).
+
+-   Introduce compact tuples that allow saving 4 bytes per tuple in case of small userdata
```

Comment:
```suggestion
-   Introduced compact tuples that allow saving 4 bytes per tuple in case of small userdata
```
keeping verb tense consistent across bullet points is a good practice :)